deploy/k8s/upload-sgx.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: upload
+  namespace: faasm
+  labels:
+    app: faasm
+    role: upload
+spec:
+  containers:
+    - name: upload
+      image: faasm/upload:0.8.3
+      ports:
+        - containerPort: 8002
+        - containerPort: 5000
+      env:
+        - name: REDIS_STATE_HOST
+          value: redis-state
+        - name: REDIS_QUEUE_HOST
+          value: redis-queue
+        - name: LOG_LEVEL
+          value: "info"
+        - name: LD_LIBRARY_PATH
+          value: "/build/faasm/third-party/lib:/usr/local/lib"
+        - name: WASM_VM
```

Comment:
This file is a copy of `deploy/k8s/upload.yml` with just one extra environment variable.